R/survival.R

```diff
@@ -374,25 +383,25 @@ surv_juv_outmigration_san_joaquin <- function(..surv_juv_outmigration_sj_int = -
 #' @param avg_temp Variable describing monthly mean temperature in celsius
 #' @param perc_diversions Variable describing monthly mean percent diverted
-#' @param .intercept_one Intercept for model one, source: TODO
-#' @param .intercept_two Intercept for model two, source: TODO
-#' @param .intercept_three Intercept for model three, source: TODO
-#' @param .delta_flow Coefficient for delta_flow variable, source: TODO
-#' @param .avg_temp Coefficient for avg_temp variable, source: TODO
-#' @param .perc_diversions Coefficient for perc_diversions variable, source: TODO
-#' @param .medium parameter for medium sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
-#' @param .large parameter for large sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
+#' @param .intercept_one Intercept for model one
+#' @param .intercept_two Intercept for model two
+#' @param .intercept_three Intercept for model three
+#' @param .delta_flow Coefficient for delta_flow variable
+#' @param .avg_temp Coefficient for avg_temp variable
+#' @param .perc_diversions Coefficient for perc_diversions variable
+#' @param .medium parameter for medium sized fish
+#' @param .large parameter for large sized fish
 #' @source IP-117068
 #' @export
 surv_juv_outmigration_sac_delta <- function(delta_flow, avg_temp, perc_diversions,
-                                            .intercept_one = -3.5,
-                                            .intercept_two =  0.3,
-                                            .intercept_three = -3.5,
-                                            .delta_flow = 0.0013,
-                                            .avg_temp = 0.386,
-                                            .perc_diversions = -0.033,
-                                            .medium = 1.48,
-                                            .large = 2.223){
+                                            .intercept_one = fallRunDSM::params$.surv_juv_outmigration_sac_delta_intercept_one,
+                                            .intercept_two = fallRunDSM::params$.surv_juv_outmigration_sac_delta_intercept_two,
+                                            .intercept_three = fallRunDSM::params$.surv_juv_outmigration_sac_delta_intercept_three,
+                                            .delta_flow = fallRunDSM::params$.surv_juv_outmigration_sac_delta_delta_flow,
+                                            .avg_temp = fallRunDSM::params$.surv_juv_outmigration_sac_delta_avg_temp,
+                                            .perc_diversions = fallRunDSM::params$.surv_juv_outmigration_sac_delta_perc_diversions,
+                                            .medium = fallRunDSM::params$.surv_juv_outmigration_sac_delta_medium,
+                                            .large = fallRunDSM::params$.surv_juv_outmigration_sac_delta_large){
 
   model_weight <- 1/3
```

Comment:
`model_weight` needs to be lifted for sensitivity analysis. I don't know if this is implemented correctly either. Flagged for @ergz and I to fix